### ISSUE TITLE
Build error : Cannot find type 'SFSymbol' in scope

### ISSUE THREAD

**Issue by inzan** [SwiftKickMobile/SwiftRepo#1]
Hi, this looked like an interesting package. I tried using it in a sample app but I get the following error when trying to build after adding the package:

UIError.swift:26

"Cannot find type 'SFSymbol' in scope"

Good luck with this and thanks for any info you can provide with this issue.

**Comment by wtmoose** [comment]
Yeah, we’re still merging some changes. Check back tomorrow. However, we’re made this public for easier distribution to clients and aren’t planning to write up a readme or make a sample project at this time

**Comment by wtmoose** [comment]
Closing this as fixed by #2. I'm not tagging an initial release just yet. We're going to be adding more SwiftData integrations next week and some types are going to get renamed. Once those changes are in, there will be a release.
